src/mgmt/srv_target.c

```diff
@@ -1075,6 +1087,14 @@ ds_mgmt_hdlr_tgt_create(crt_rpc_t *tc_req)
 	if (tca.tca_newborn != NULL) {
 		struct vos_pool_arg vpa = {0};
 
+		/* clear any orphaned blobs that are associated with this uuid */
```

Comment:
No change requested necessarily - just adding some analysis here. The biggest thing I'd like to be sure of is that a retry MGMT_TGT_CREATE corpc will not clean up blobs created in the earlier handling. It seems that the first handling will have created the files in the NEWBORNS directory in "SCM" so in the retry handler tca.tca_newborn == NULL. So I think this will successfully avoid clear_vos_pool()

In a separate chat we had also considered another possible avenue of implementation to update the MGMT_TGT_CREATE cart RPC definition to have a boolean to ask for the clearing to occur. And it could be engaged just by pool reintegration / as opposed to pool create execution flows. But I think settled on the current implementation approach as preferred.